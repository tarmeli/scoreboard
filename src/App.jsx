import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import Routes from "./routes/Routes";
import Navigation from "./components/Navigation/Navigation";
import PlayersProvider from "./data/context/players/PlayersProvider";

const App = () => (
    <Router>
        <Navigation />

        <PlayersProvider>
            <Routes />
        </PlayersProvider>
    </Router>
);

export default App;
