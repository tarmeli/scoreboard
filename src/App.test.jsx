import React, { useState as useStateMock } from "react";
import { mount } from "enzyme";

import App from "./App";

describe("App", () => {
  let setStateMock;
  let component;

  beforeEach(() => {
    setStateMock = jest.fn();
    useStateMock.mockImplementation((init) => [init, setStateMock]);

    component = mount(<App />);
  });

  it("renders", () => {
    expect(component.render()).toMatchSnapshot();
  });
});
