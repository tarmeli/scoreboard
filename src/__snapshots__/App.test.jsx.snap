// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
Array [
  <nav>
    <ul>
      <li>
        <a
          alt="Houston table tennis scoreboard"
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          alt="Players page"
          href="/players"
        >
          Players
        </a>
      </li>
    </ul>
  </nav>,
  <div>
    Hi
  </div>,
]
`;
